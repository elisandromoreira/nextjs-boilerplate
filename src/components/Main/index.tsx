const Main = () => (
  <main>
    <h1>NextJS Boilerplate</h1>
  </main>
);

export default Main;
