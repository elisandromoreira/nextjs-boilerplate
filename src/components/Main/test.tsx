import { render, screen } from '@testing-library/react';

import Main from '.';

describe('<Main />', () => {
  it('shoud render the heading', () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole('heading', { name: /nextjs boilerplate/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
