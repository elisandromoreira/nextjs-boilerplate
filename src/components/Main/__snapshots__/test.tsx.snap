// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> shoud render the heading 1`] = `
<main>
  <h1>
    NextJS Boilerplate
  </h1>
</main>
`;
